src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/restapi/controller/ReferralController.kt

```diff
@@ -83,4 +86,27 @@ constructor(
       ),
     )
   }
+
+  override fun getReferralSummariesByUsername(
+    @RequestParam(value = "page", defaultValue = "0") page: Int,
+    @RequestParam(value = "size", defaultValue = "10") size: Int,
+    @RequestParam(value = "status", required = false) status: List<String>?,
+    @RequestParam(value = "audience", required = false) audience: String?,
+    @RequestParam(value = "courseName", required = false) courseName: String?,
+  ): ResponseEntity<PaginatedReferralSummary> {
+    val pageable = PageRequest.of(page, size, Sort.by("referralId"))
+    val username: String = securityService.getCurrentUserName() ?: throw AccessDeniedException("unauthorised, username not present in token")
```

Comment:
We might want to move this into the `securityService` in future, but I think this is ok for now.